content/release-10-18-0/digital-twin-manager-10-18-0-bundle/10-18-0-1.md

```diff
@@ -0,0 +1,355 @@
+---
+weight: 40
+title: Release 10.18.0.1
+layout: bundle
+---
+
+### Improvements
+
+<div><table ><colgroup>
+<col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Description</th>
+<th>
+Issue</th>
+<th>
+Build version</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>The columns **Child asset models** and **Asset properties** have been added to the table in the <b>Asset models</b> page.</td>
```

Comment:
@MWindel How about the formatting of column names in the DTM documents? Are they formatted bold?

Reply:
Yes they are in Bold

Reply:
@spau6 Then you must use <b> instead of **.